app/controllers/get_involved_controller.rb

```diff
@@ -23,9 +23,9 @@ def load_content_item
   end
 
   def load_get_involved_data
-    @open_consultation_count = retrieve_given_document_type("open_consultation")['total']
+    @open_consultation_count = Services.search_api.search({ filter_display_type: "Open consultation", count: 0 })["total"]
```

Comment:
Could this be filtered on a static key rather than text that could change?

I think [`filter_content_store_document_type: "open_consultation"`](https://www.gov.uk/api/search.json?filter_content_store_document_type=open_consultation) should suffice.

Same further down each time this query is being made.